organization of some rules and elimination of useless

mefsaal opened the pull request.

mefsaal committed:
add gitignore

mefsaal committed:
upd gitignore

mefsaal committed:
make some changes to types in ananicy

TODO: separate deduplication programs and other tools

mefsaal committed:
Move deprecated rules to disabled for best organization

mefsaal committed:
this doesnt exist

mefsaal committed:
my mistake

mefsaal committed:
greater organization of the rules and transfer to the corresponding directories

Technetium1 reviewed (approved):
This all seems good :eyes:

mefsaal committed:
Minq-Ananicy Not Ananicy

only a name change noting more

mefsaal committed:
im stupid

mefsaal committed:
little fix

mefsaal commented:
still need some tests, if anyone can do it. Welcome (can't test all the rules and types, sorry)

mefsaal committed:
fix

kuche1 commented on ananicy.d/00-types.types:
```diff
@@ -90,7 +94,7 @@
 {"type":"VPN"}
 
-{"type":"VM"}
+{"type":"VM,CPU_H,IO_H",    "ionice":7}
```
The `_H` postfix stands for higher priority. If we are to create a new type called `VM,CPU_H,IO_H`, that would imply that IO nice is about 0, and that nice is about -20.
If are to keep the `"ionice":7`, then we would need to change the type to `VM,CPU_U,IO_L` or `VM,IO_L`.

mefsaal replied:
I don't know, I didn't see documentation that says anything about

kuche1 commented on .gitignore:
```diff
@@ -0,0 +1,2 @@
+.vscode/settings.json
+.vscode/
```
This needs to be removed.

mefsaal replied:
Yes

kuche1 reviewed (changes requested):
Will merge after the 2 comments get fixed.

mefsaal committed:
upd

mefsaal committed:
organization for some  types and more

I need more testers

mefsaal commented:
I moved a couple of rules to group them and make it easier to recognize them. I changed the IO/CPU on some and did a couple of tests.

I need testers, unfortunately I can't test all the rules on my pc/environment

mefsaal commented:
I think is stable @kuche1 Plz merge

Technetium1 commented:
Apologies for the delayed responses, life is VERY busy right now, I haven't been able to review the full changes since last.

Why is this changed to a `TODO` now?
```
{"name": "kbfsfuse", "type": "TODO"}
```
